### PR TITLE
Updates for week of Jan 17-23

### PR DESCRIPTION
**Pull request by halocline**
#### What does this PR do?

Adds What's New updates for the week of Jan 11-23

[Preview](https://6010b0e756381600071217f2--keen-mayer-a86c8b.netlify.app/whats-new)

### CODE REVIEW

File: aries-site/src/pages/whats-new.mdx [hunk 1]
```diff
@@ -53,6 +53,39 @@ either via [Slack](https://hpe.slack.com/archives/C04LMJWQT).
...
+  - Added an example and guidance for using the [basic layout with cards](https://design-system.hpe.design/foundation/background-colors-guidance#basic-layout-with-cards).
+  - Added examples clarifying how an application header should appear and behave when it is a [fixed header](https://design-system.hpe.design/foundation/background-colors-guidance#with-fixed-header) and another for when the [header scrolls with the content](https://design-system.hpe.design/foundation/background-colors-guidance#header-scrolls-with-content).
+  
+### InfiniteScroll
+
+- Completed [InfiniteScroll design direction](https://www.figma.com/file/XOEwUOqtfLAowprV2psWHM/HPE-Infinite-Scroll?node-id=1%3A35).
+  
+### Layer
+
+- Enhanced ability for layers to have greater control over how [layer corner rounding](https://github.com/grommet/grommet/pull/4888) is applied.
```
Comment (taysea) [inline aries-site/src/pages/whats-new.mdx:71]: Need to apply the new default to HPE theme. Will file a PR today. However, all grommet functionality is in.

Comment (halocline) [reply aries-site/src/pages/whats-new.mdx:71]: Want to wait for this one until next week? Want to rephrase?

**Comment by halocline** [comment]
@taysea Can you also peek at the Applitools snapshots? To me it looks like we have diffs related to a timing issue for when/how snapshots are taken on Windows10/Edge where the snapshot is taken before the viewport width has been adjusted. Manual review on Edge looks good to me.

File: aries-site/src/pages/whats-new.mdx [hunk 2]
```diff
@@ -53,6 +53,39 @@ either via [Slack](https://hpe.slack.com/archives/C04LMJWQT).
 
 <!-- BEGIN: What's New Entries -->
 
+## January 17-23, 2021
+
+### Background colors
+
+- Incorporated feedback and addressed clarifying questions:
+  - Provided additional guidance what to consider when [selecting a basic or layered approach](https://design-system.hpe.design/foundation/background-colors-guidance#basic-or-layered-approach).
```
Comment (ShimiSun) [inline aries-site/src/pages/whats-new.mdx:61]:
```suggestion
  - Provided additional guidance on what to consider when [selecting a basic or layered approach](https://design-system.hpe.design/foundation/background-colors-guidance#basic-or-layered-approach).
```

File: aries-site/src/pages/whats-new.mdx [hunk 3]
```diff
@@ -53,6 +53,39 @@ either via [Slack](https://hpe.slack.com/archives/C04LMJWQT).
...
+
+- Completed [InfiniteScroll design direction](https://www.figma.com/file/XOEwUOqtfLAowprV2psWHM/HPE-Infinite-Scroll?node-id=1%3A35).
+  
+### Layer
+
+- Enhanced ability for layers to have greater control over how [layer corner rounding](https://github.com/grommet/grommet/pull/4888) is applied.
+
+### Pagination
+
+- Incorporated review feedback and incorporated t-shirt sizing. [Play with Pagination behavior](https://deploy-preview-4646--sad-tereshkova-173d07.netlify.app/?path=/story/controls-pagination--custom)
```
Comment (ShimiSun) [inline aries-site/src/pages/whats-new.mdx:75]: Might be worth mentioning that is not completed and still a work in progress? Unless you think it is clear.
For example, something like:
```suggestion
- Incorporated review feedback and incorporated t-shirt sizing. [Play with Pagination POC behavior](https://deploy-preview-4646--sad-tereshkova-173d07.netlify.app/?path=/story/controls-pagination--custom) 
```

**Review by ShimiSun — COMMENTED** [review]
🎉

File: aries-site/src/pages/whats-new.mdx (hunk 1 above)
Comment (taysea) [reply aries-site/src/pages/whats-new.mdx:71]: Let's wait until next week then. Don't want to cause confusion if it's not in the theme yet.

**Comment by vavalos5** [comment]
Looks good! I'm thinking we can be open and mention that we have completed the designs to give the home page a refreshing new look and development is next.

Also adding that designer guidance and developer guidance is now easier to find.

-HPE Design system video is also included on the home page

**Review by vavalos5 — COMMENTED** [review]
Looks good! I'm thinking we can be open and mention that we have completed the designs to give the home page a refreshing new look and development is next.

Also adding that designer guidance and developer guidance is now easier to find.

-HPE Design system video is also included on the home page

File: aries-site/src/pages/whats-new.mdx (hunk 3 above)
Comment (halocline) [reply aries-site/src/pages/whats-new.mdx:75]: Good call. Yes, that could be made more clear.

**Comment by halocline** [comment]
> Looks good! I'm thinking we can be open and mention that we have completed the designs to give the home page a refreshing new look and development is next.
> 
> Also adding that designer guidance and developer guidance is now easier to find.
> 
> -HPE Design system video is also included on the home page

@vavalos5 Can we touch base after standup? I want to make sure we get this in how you like.

File: aries-site/src/pages/whats-new.mdx (hunk 3 above)
Comment (halocline) [reply aries-site/src/pages/whats-new.mdx]: Done.

File: aries-site/src/pages/whats-new.mdx (hunk 1 above)
Comment (halocline) [reply aries-site/src/pages/whats-new.mdx]: Done.

File: aries-site/src/pages/whats-new.mdx (hunk 2 above)
Comment (halocline) [reply aries-site/src/pages/whats-new.mdx]: Done.

**Pull request merged**